lib/src/lifecycle_module.dart

```diff
@@ -67,9 +67,9 @@ abstract class LifecycleModule {
   /// Calls the onLoad() method, which can be implemented on a Module.
   /// Executes the willLoad and didLoad event streams.
   Future load() async {
-    _willLoadController.add(this);
+    await _willLoadController.add(this);
     await onLoad();
-    _didLoadController.add(this);
+    await _didLoadController.add(this);
```

Comment:
it seemed in the prior release of dart that the `load()` future wouldn't resolve itself until all of the events had been propagated to the streams and listeners, where as after 1.13.0 the future would resolve itself prior the listener actually receiving the data